runtime/doc/options.txt

```diff
@@ -1087,7 +1087,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 <	Use 'backupdir' to put the backup in a different directory.
 
 						*'backupskip'* *'bsk'*
-'backupskip' 'bsk'	string	(default: "/tmp/*,$TMPDIR/*,$TMP/*,$TEMP/*")
+'backupskip' 'bsk'	string	(default: "$TMPDIR/*,$TMP/*,$TEMP/*",
+				 Unix: "/tmp/*,$TMPDIR/*,$TMP/*,$TEMP/*",
+				 Mac OS X:
+				 "/private/tmp/*,$TMPDIR/*,$TMP/*,$TEMP/*")
```

Comment:
Please explain why you made more changes than adding the macOS entry.  Even if the changes appear to be more consistent with the description of `:h bsk`, people probably want to know why the value of `default` was changed and the new entry `Unix` was introduced.

Or,

1. Remove the whole line of `default`,
2. Make `Unix` revert to `default`, and
3. Change `Mac OS X` to `macOS`.

Reply:
Ah, okay.  Bram already approved the changes.  Since I've been away from the keyboard since yesterday evening until now, I didn't notice the patch has already been included.  Sorry about the noise.